module/icp/io/aes.c

```diff
@@ -92,11 +92,6 @@ static crypto_mech_info_t aes_mech_info_tab[] = {
 	    AES_MIN_KEY_BYTES, AES_MAX_KEY_BYTES, CRYPTO_KEYSIZE_UNIT_IN_BYTES}
 };
 
-/* operations are in-place if the output buffer is NULL */
-#define	AES_ARG_INPLACE(input, output)				\
-	if ((output) == NULL)					\
-		(output) = (input);
-
```

Comment:
On a second thought, wouldn't it be better to just change this macro to `ASSERT(output != NULL)`? This would be more in line with the other asserts.

Reply:
@AttilaFueloep Updated to switch to this assertion instead.

Reply:
@dbussink Thanks for doing the changes. This looks good to me now.